### ISSUE TITLE
SyntaxError: Unexpected token export

### ISSUE THREAD

**Issue by Roseidon** [markgab/json-porter#1]
I use the json-porter in a typescript project and use it as described in the introduction.
Everything works fine when running the project, but when running tests with `nyc mocha`, I encounter the error
```
…/node_modules/json-porter/json-porter.js:117
export default JsonPorter;
^^^^^^

SyntaxError: Unexpected token export
    at Module._compile (internal/modules/cjs/loader.js:723:23)

```

Has anyone ever encountered this error before? It seems that some of my configuration for mocha is wrong, I think.

**Comment by Roseidon** [comment]
Investigating a little further, running TypeScript before the tests, seemed to solve this issue but uncovered several other issues in my code. I better keep TypeScript off :D

**Comment by markgab** [comment]
@Roseidon, thanks for the feedback. I suspect I should be compiling the TypeScript to an earlier ES version of JavaScript to avoid this issue. I will look into doing this with the next release. Other than the issue with unit testing, did it work well for you?

**Comment by markgab** [comment]
@Roseidon, I just published version 1.0.4 which compiles modules to commonjs. I believe this should resolve your error. Please confirm.

Thanks!

**Comment by Roseidon** [comment]
Besides the issue with testing, exporting and importing files worked fine.

I just tried running the tests, everything works now with version 1.0.4

Thank you for the quick support! :)
